docs/faq/general/how-do-i-allowlist-codacy-on-my-git-provider.md

```diff
@@ -17,7 +17,7 @@ To allowlist Codacy Cloud on your Git provider:
 1.  Send an email to <span class="skip-vale">[success@codacy.com](mailto:support@codacy.com?subject=Enabling static IP addresses)</span> or directly to your CSM asking us to enable static IP addresses for your organization.
```

Comment:
the email in the text is different from the email on the link success -> support

Reply:
Well spotted! I'm fixing it now.